packages/shared/src/components/widgets/PostUsersHighlights.tsx

```diff
@@ -88,12 +94,25 @@ const UserHighlight = (props: SourceAuthorProps) => {
     reputation,
   } = props;
   const Icon = getUserIcon(userType);
-  const userTypeNotSource = userType !== 'source';
+  const isUserTypeSource = userType === 'source';
```

Comment:
Should we add an enum for userType and use it instead of a string value here?

Reply:
Its a local type for a component. I can do enum yes.